packages/react-components/review/beta/react-components.api.md

```diff
@@ -234,14 +234,21 @@ export interface CameraSitePermissionsProps extends CommonSitePermissionsProps {
 // @beta
 export type CameraSitePermissionsStrings = SitePermissionsStrings;
 
+// @internal
+export const _Caption: (props: _CaptionProps) => JSX.Element;
+
+// @internal
+export interface _CaptionProps extends _CaptionsInfo {
+    onRenderAvatar?: OnRenderAvatarCallback;
+}
+
 // @internal
 export const _CaptionsBanner: (props: _CaptionsBannerProps) => JSX.Element;
 
 // @internal
 export interface _CaptionsBannerProps {
```

Comment:
onRenderAvatar?: OnRenderAvatarCallback; should still stay here, as well as CaptionsInfo,
Then you can construct _CaptionProps in the CaptionsBanner for Caption component